modules/apache-airflow/templates/values.yaml

```diff
@@ -2037,13 +2037,13 @@ cleanup:
   # Labels specific to cleanup objects and pods
   labels: {}
 
-  resources: {}
-  #  limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  #  requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+  resources:
+   limits:
+    cpu: 100m
+    memory: 128Mi
+   requests:
+    cpu: 100m
+    memory: 128Mi
```

Comment:
These resource allocations seem pretty minimal to me. Is that accurate, or am I off base there? Is there any reason to bump it up?

Reply:
This was their recommended defaults. We can adjust these later on by looking at historical data in grafana.